extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -14,335 +14,394 @@
+   transition: opacity 300ms linear!important;
+ }
+ 
+ .i-amphtml-story-share-pill:hover .i-amphtml-story-share-item {
+   opacity: 1!important;
+ }
+ 
+ .i-amphtml-story-share-pill .i-amphtml-story-share-item {
+   margin: 0 !important;
+   width: 48px !important;
+   height: 48px !important;
+   padding: 0 !important;
+ }
+ 
+ /* Incremental fade in */
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(9) {
+   transition-delay: calc(30ms * 1)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(8) {
+   transition-delay: calc(30ms * 2)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(7) {
+   transition-delay: calc(30ms * 3)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(6) {
+   transition-delay: calc(30ms * 4)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(5) {
+   transition-delay: calc(30ms * 5)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(4) {
+   transition-delay: calc(30ms * 6)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(3) {
+   transition-delay: calc(30ms * 7)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(2) {
+   transition-delay: calc(30ms * 8)!important;
+ }
+ .i-amphtml-story-share-pill:hover
+     .i-amphtml-story-share-item:nth-child(1) {
+   transition-delay: calc(30ms * 9)!important;
+ }
+ 
+ .i-amphtml-story-share-pill .i-amphtml-story-share-list {
+   padding: 0!important;
+   margin: 0 72px 0 0 !important;
+ }
+ 
+ .i-amphtml-story-share-pill:hover:before {
+   width: calc(100% - 32px) !important;
+ }
+ 
+ .i-amphtml-story-share-pill .i-amphtml-story-share-icon {
+   display: flex !important;
+   justify-content: center;
+   align-items: center;
+   width: 48px !important;
+   height: 48px !important;
+   padding: 0;
+ }
+ 
+ [desktop] .i-amphtml-story-share-icon {
+   font-size: 0px!important;
+   background-size: 24px 24px !important;
+   background-position: center center !important;
+   padding: 0 !important;
+ }
+ 
+ [desktop] .i-amphtml-story-share-icon.amp-social-share-facebook {
+   /* SVG icon has inner padding, so the glyph renders visually smaller than
+    * other icons, so we work around it by setting a higher background size. */
+   background-size: 28px 28px !important;
+   background-position: center center !important;
+ }
+ 
+ [desktop] .i-amphtml-story-share-icon::before {
+   display: none !important;
+ }
+ 
+ [desktop] .i-amphtml-story-share-icon:active::before {
+   display: none !important;
+ }
+ 
+ [desktop] .i-amphtml-story-share-list {
+   margin-right: 72px !important;
+ }
+ 
+ /* system layer changes */
+ 
+ [desktop] .i-amphtml-story-share-pill-container {
+   z-index: 100002 !important; /* 1 higher than bookend */
+ }
```

Comment:
nit: leave newline at end of file

see: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files